test/cli_run_pid_test.go

```diff
@@ -55,9 +57,16 @@ func (suite *PouchRunPidSuite) TestRunWithPidsLimit(c *check.C) {
 	})
 
 	cname := "TestRunWithPidsLimit"
-	pidfile := "/sys/fs/cgroup/pids/pids.max"
-	res := command.PouchRun("run", "--pids-limit", "10",
-		"--name", cname, busyboxImage, "cat", pidfile)
+	res := command.PouchRun("run", "-d", "--pids-limit", "10",
+		"--name", cname, busyboxImage, "top")
+	res.Assert(c, icmd.Success)
+
+	cid := strings.TrimSpace(res.Stdout())
+	pidfile := fmt.Sprintf(
+		"/sys/fs/cgroup/pids/default/%s/pids.max",
```

Comment:
I don't think we can see all the containers cgroup in one container.

Reply:
I am afraid this command is executed on the local machine, not in the container.

Reply:
I agree with @fuweid .
"/sys/fs/cgroup/pids/default/" should be seen in a container.

Currently, this test is pass in 4.9 kernel. If the pr is merged, this test would be failed in 4.9 kernel.

Reply:
The v2 patch only checks whether pid cgroup work from local host,  because there isn't /sys/fs/cgroup/pids/pids.max file in container in AliOS3.10 kernel